common/src/main/java/com/wynntils/features/combat/HorseMountFeature.java

```diff
@@ -105,6 +124,11 @@ private void trySummonAndMountHorse(int horseInventorySlot, int attempts) {
         if (attempts <= 0) {
             postHorseErrorMessage(MountHorseStatus.NO_HORSE);
             return;
+        } else if (cancelMountingHorse != null) {
+            McUtils.sendPacket(new ServerboundSetCarriedItemPacket(prevItem));
+            postHorseErrorMessage(cancelMountingHorse);
+            cancelMountingHorse = null;
+            return;
```

Comment:
The else is not needed.

Secondly, doesn't this duplicate messages? Since you redirect it in `ChatRedirectFeature` and post the error messages here too? I am not sure what the cleanest solution would be, but perhaps leaving it to `ChatRedirectFeature` and not posting errors in this case might be the best (but still parsing the errors of course).

Reply:
Yeah it does. We can leave it, but if the player has the redirect for horse off, the error messages when the horse can't be found by us and the messages from wynn will be inconsistent. I'll remove it anyway

Reply:
Well, yeah, that is on them. But if they turn off `GameNotificationOverlayFeature` all of it will go to the chat. So I think it is fine.